lib/runner/mocha-runner/skip/skip-builder.js

```diff
@@ -17,20 +17,27 @@ module.exports = class SkipBuilder {
         this._currentBrowserId = browserId;
     }
 
-    in(matcher, comment) {
-        return this._processSkip(matcher, comment);
+    in(matcher, comment, opts) {
+        return this._processSkip(matcher, comment, opts);
     }
 
-    notIn(matcher, comment) {
-        return this._processSkip(matcher, comment, {negate: true});
+    notIn(matcher, comment, opts) {
+        opts = _.extend(opts || {}, {negate: true});
+        return this._processSkip(matcher, comment, opts);
+    }
+
+    only() {
```

Comment:
`only` должен быть не у `skip`, а отдельный объект
Вот эти записи должны быть эквивалентны:
```js
hermione.skip.notIn('ie10', 'Because we can', {silent: true});
hermione.only.in('ie10');
```